easybuild/easyconfigs/c/charmm/charmm-43b2-foss-2017b.eb

```diff
@@ -0,0 +1,30 @@
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+easyblock = 'EB_CHARMM'
+
+name = 'charmm'
```

Comment:
@vanzod Why `charmm` rather than `CHARMM` (cfr. description)?

Reply:
`charmm` is the non-free version of `CHARMM`. I have added a line in the description to highlight the difference.